dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -99,12 +99,13 @@ def load_level_source
 
     return unless current_user
 
-    if params[:load_previous]
-      @start_blocks = current_user.last_attempt(@level).try(:level_source).try(:data)
-    elsif params[:level_source_id]
+    if params[:level_source_id]
       level_source = LevelSource.find(params[:level_source_id])
       # we do multiple level projects, so we don't check that the level_source.level_id matches the loaded level
       @start_blocks = level_source.data
+    elsif current_user.try(:admin?)
+      last_attempt = current_user.last_attempt(@level).try(:level_source).try(:data)
+      @start_blocks = last_attempt unless last_attempt.nil?
```

Comment:
nit: I would probably do this as `@start_blocks = last_attempt if last_attempt` to avoid the double negative